tests/helpers/package-cache.js

```diff
@@ -3,20 +3,18 @@
 var fs = require('fs-extra');
 var path = require('path');
 var quickTemp = require('quick-temp');
-var merge = require('ember-cli-lodash-subset').merge;
 var Configstore = require('configstore');
 var CommandGenerator = require('./command-generator');
 var stableStringify = require('json-stable-stringify');
 
-// This is the "root" folder.
-var emberCLIPath = path.resolve(__dirname, '../..');
+var originalWorkingDirectory = process.cwd();
 
 // Module scoped variable to store whether a particular cache has been
 // attempted to be upgraded.
 var upgraded = {};
 
-// Module scoped variable set on first Ember CLI link attempt per type.
-var isEmberCLILinked = {};
+// List of keys which can result in things being installed.
+var DEPENDENCY_KEYS = ['dependencies', 'devDependencies', 'peerDependencies', 'optionalDependencies'];
```

Comment:
This array of items was already present in `_checkManifest`, now extracted as a "`const`". It's intended to be the complete set of things which could _possibly_ result in the package manager installing something. This is the "err on the side of caution" approach. It actually doesn't matter if something is or isn't automatically installed in any of the cases where we use this.